frontends/concrete-python/tests/execution/test_tfhers.py

```diff
@@ -499,7 +497,21 @@ def test_tfhers_client_specs(function, parameters, dtype: tfhers.TFHERSIntegerTy
         ],
     )
 
-    client = circuit.client
+    # save and load server
+    via_mlir = np.random.choice([True, False])
+    _, server_path = tempfile.mkstemp(suffix=".zip")
+    circuit.server.save(server_path, via_mlir=via_mlir)
+    server = fhe.Server.load(server_path, via_mlir=via_mlir)
+    os.remove(server_path)
+    assert isinstance(server.client_specs.tfhers_specs, tfhers.TFHERSClientSpecs)
```

Comment:
please test bridge creation here!

Reply:
It would be more about testing the creation of a Client from a Server which is already tested elsewhere. Here we are testing that tfhers_specs is well saved/loaded in all scenario.